SPEC/REPO.md

```diff
@@ -20,7 +20,7 @@ Where:
 - `err` is an Error if the whole GC operation was not successful.
 - `res` is an array of objects that contains the following properties
   - `err` is an Error if it was not possible to GC a particular block.
-  - `cid` is the [CID][cid] of the block that was Garbage Collected.
+  - `multihash` is the [multihashes][multihashes] of the block that was Garbage Collected.
```

Comment:
If the text in the square brackets is the same as the shortcut in the footer, I think this can just be `[multihashes][]`

Reply:
Oh did not know that! That is neat! Thanks for the heads up!